app/posts/2021-06-15-hiring-staff-update.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Improvements to the hiring staff dashboard
+description: Introducing job pages into our information architecture for hiring staff.
+date: 2021-06-15
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+We’ve recently introduced the ability for people to apply for jobs in our service and for hiring staff to manage these applications. This has significantly increased the complexity of what _a job_ is within our service. 
+
+We realised that our current interface would not be able to scale to allow hiring staff to complete the following tasks in a consistent location: 
+
+* manage jobs (delete/edit/copy/extend deadlines)
+* collect and manage applications (view/shortlist/reject)
+* view a job’s statistics (views/saves etc)
+* view diversity data collected in the application process
+
+## Updating the jobs dashboard
+
+### The problem
+
+Previously users accessed all job related tasks from their main jobs dashboard, and managed applications for a job in a separate screen.
+
+{{ appFigure({
+  image: {
+    file: "old-job-view.png",
+    alt: "A job in the hiring staff dasboard"
+  }
+}) }}
+
+All text within each row on the jobs dashboards were all a similar size, this lack of typographic hierarchy made it hard to establish an order of importance within the content. Some links were actions (for example, ‘edit listing’ or ‘extend deadline’) and others were navigation based (for example the titles of the jobs),. This made it difficult for the user to distinguish what the purpose of each link was.
```

Comment:
The wording of this sentence doesn't quite work, perhaps the second "all" can be removed?

Reply:
Good pick up!